README.md

```diff
@@ -1,2 +1,34 @@
-# MobilePortfolio
-Microversepro
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+>Module 1 project 
+
+
+## Built With
+
+- Html
+- Css
+- git and Github
+
+
+
+
+
+
+
+
+
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@Xhibitbrown](https://github.com/githubhandle)
+- Twitter: [@AdubofourB](https://twitter.com/twitterhandle)
+- LinkedIn: [Adubofour Kusi Brown](https://linkedin.com/in/linkedinhandle)
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
- [ ] Kindly make sure you update your README file by completing all the important sections from the **[Microverse README template](https://github.com/microverseinc/readme-template)**. Your README should contain **a project title, a good and meaningful project description, Getting started section(with information about setting up the project, cloning the repo, accessing the cloned directory,...)**. I suggest you take a look at **[this template](https://github.com/tresorsawasawa/MyPortfolio/blob/develop/README.md)** to help you fill all the sections.